site/src/pages/components/accordion.explainer.mdx

```diff
@@ -7,9 +7,15 @@ layout: ../../layouts/ComponentLayout.astro
 ---
 
 - [@dbaron](https://github.com/dbaron)
-- Last updated: September 11, 2023
-- Feedback: https://github.com/openui/open-ui/issues/725
-- HTML specification pull request: https://github.com/whatwg/html/pull/9400
+- Last updated: November 16, 2023
+- Github discussions:
+  - https://github.com/openui/open-ui/issues/725
```

Comment:
Nit: it's always nice to label these with a quick one-liner so it's easy to see what each one is about. Not required though.

Reply:
OK, labeled them a bit better.